common/src/main/java/com/wynntils/utils/type/BoundingBox.java

```diff
@@ -31,6 +31,6 @@ public boolean contains(float x, float z) {
     public boolean intersects(BoundingBox other) {
         boolean xIntersects = Math.max(x1, other.x1) < Math.min(x2, other.x2);
         boolean zIntersects = Math.max(z1, other.z1) < Math.min(z2, other.z2);
-        return xIntersects && zIntersects;
+        return xIntersects || zIntersects;
```

Comment:
I saw the bug with the minimap recently, but are you sure this is the correct fix? It seems like that when I take an intersection of 2 2d objects, I need to have intersections on both axes.

Reply:
Actually I just tried it again and it is rendering pois outside of the area with this change so no I don't think it actually is correct, I'll revert that